config/application.rb

```diff
@@ -45,13 +45,17 @@ class Application < Rails::Application
       config.x.send("#{key}=".to_sym, value)
     end
 
-    # Set default_url_options for ActionMailer
+    # Set default_url_options and default from for ActionMailer
     if config.x.hostname.present? && config.x.port.present?
       config.action_mailer.default_url_options = { host: config.x.hostname,
                                                    port: config.x.port }
+      from = "do-not-reply@#{config.x.hostname}:#{config.x.port}"
+      ActionMailer::Base.default from: from
```

Comment:
In the Devise config we're using `Rails.configuration.x.from_email` (from general.yml), can we do the same here?